tests/tests/test-starknet/test-starknet-extrinsics.ts

```diff
@@ -99,4 +105,26 @@ describeDevMadara("Pallet Starknet - Extrinsics", (context) => {
       )
     ).to.exist;
   });
+
+  it("deploys ERC20 contract via UDC", async function () {
```

Comment:
As discussed earlier in the chat, we need a separate issue to improve all extrinsic testing in this PR, { I am making that and will take that issue as well. }

For now, this extrinsic test will depend on "ExitinsicSuccess" like all other tests in this file, and then I will make a new PR with a separate issue, changing all extrinsic tests that reside in this file.